app/models/comment.rb

```diff
@@ -1,4 +1,6 @@
 class Comment < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ]  Great job so far but I think it would be great if you add here reference to the class with the foregin key so I recommend to add it here for the three relation since in this comment table there is author_id reference to it 🤩✅ so please add it

### You can do somthing like this ⬇
`belongs_to :author, class_name: 'User'`